stories/features/rewards/settings/settingsPage.tsx

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+// Components
+import AdsBox from './adsBox'
+import ContributeBox from './contributeBox'
+import DonationsBox from './donationsBox'
+import Grant from './grant'
+import PageWallet from './pageWallet'
+import { Grid, Column } from '../../../../src/components'
+import { MainToggle, SettingsPage } from '../../../../src/features/rewards'
+
+// Assets
+import '../../../assets/fonts/muli.css'
+import '../../../assets/fonts/poppins.css'
+
+interface State {
+  mainToggle: boolean
+}
+
+class Settings extends React.PureComponent<{}, State> {
+  constructor (props: {}) {
+    super(props)
+    this.state = {
+      mainToggle: true
+    }
+  }
+
+  onMainToggle = () => {
+    this.setState({ mainToggle: !this.state.mainToggle })
+  }
+
+  render () {
+    return (
+      <SettingsPage>
+        <Grid columns={3} theme={{ gridGap: '32px' }}>
+          <Column size={2} theme={{ justifyContent: 'center', flexWrap: 'wrap' }}>
+            <MainToggle
+              onToggle={this.onMainToggle}
+              enabled={this.state.mainToggle}
+            />
+            <AdsBox/>
+            <ContributeBox/>
+            <DonationsBox/>
+          </Column>
+          <Column size={1} theme={{ justifyContent: 'center', flexWrap: 'wrap' }}>
+            <Grant/>
+            <PageWallet/>
+          </Column>
+        </Grid>
+      </SettingsPage>
+    )
+  }
+}
+
+export default Settings
```

Comment:
Since we are importing this as `SettingsPage` let's name the export that, wdyt?

Reply:
I agree, but sadly we can't because we already have component  named like this

`import { MainToggle, SettingsPage } from '../../../../src/features/rewards'`

Reply:
maybe we can rename this file to settings.tsx?

Reply:
Sgtm

Reply:
@ryanml done